includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -145,6 +146,23 @@ public function register_get_data_route() {
 		);
 	}
 
+	/**
+	 * Register POST /tasks/completed endpoint.
+	 */
+	public function register_mark_tasks_completed_route() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/tasks/completed',
```

Comment:
I wonder if `complete` (verb) would make more sense to this endpoint. But I don't have a very strong opinion about that. WDYT?

Reply:
yeah, I kept changing it back and forth. So, changing to complete!